conf/modules.config

```diff
@@ -140,7 +140,7 @@ process {
         ]
     }
     withName: 'ENSEMBLVEP_VEP' {
-        //ext.args         ='--everything --filter_common --per_gene --total_length --offline'
+        ext.args         ='--everything --filter_common --per_gene --total_length --offline'
```

Comment:
Same as in SNV workflow. We have to leave out `--filter_common --per_gene`

Reply:
done